composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "gedmo/doctrine-extensions": "^2.3",
+        "gedmo/doctrine-extensions": "^3",
```

Comment:
Dit zou ook een `^2.3 || ^3` kunnen zijn, maar dan moeten we op project-niveau 2.3 gaan afdwingen om niet naar 3 te gaan. Dit heeft namelijk tot gevolg dat de regel `dir: '%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Tree/Entity'` aangepast moet worden.

Overkomelijk, maar wel vervelend als je het niet in de gaten hebt met een `composer update`. We hebben dit al eens eerder teruggedraaid

Fixes #34

Reply:
@7ochem

Reply:
Ik vind dit soort dingen niet vervelend, maar ik kan me voorstellen dat er collega's zijn die niet van dit soort onverwachte verrassingen houden als ze een composer update doen.

Reply:
Ja, we hebben het al een keer teruggedraaid...ik ga het aanpassen naar een `5.x` release

Reply:
Geupdate :rocket: